images/README.md

```
@@ -15,7 +15,9 @@ $ docker build -t dougbtv/multus -f images/Dockerfile .
 You may wish to deploy Multus as a daemonset, you can do so by starting with the example Daemonset shown here:
 
 ```
-$ kubectl create -f ./deployments/multus-daemonset.yml
+$ kubectl create -f ./deploy/crds/k8s.cni.cncf.io_network-attachment-definitions.yml
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)